src/main/java/net/wizardsoflua/config/GeneralConfig.java

```diff
@@ -20,9 +20,9 @@
   }
 
   /**
-   * Max. number of Lua ticks a spell can run per game tick [range: 1000 ~ 10000000, default: 10000]
+   * Max. number of Lua ticks a spell can run per game tick [range: 1000 ~ 10000000, default: 50000]
    */
-  private int luaTicksLimit = 10000;
+  private int luaTicksLimit = 50000;
```

Comment:
Why did you increase this number?

Reply:
Because it takes more than 40000 lua ticks to print a chat event like this: ```lua
print(str(event))
```
And I wanted users to be able to use such a print in an event handler.